### PR TITLE
new file:   packages/batteries/batteries.3.7.2/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**
bugfix release for ocaml-5.1.1~rc1

fix by @kit-ty-kate

### CODE REVIEW

File: packages/batteries/batteries.3.7.2/opam [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "1" {with-test}
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo:
+  "git+https://github.com/ocaml-batteries-team/batteries-included.git"
+url {
+  src: "https://github.com/ocaml-batteries-team/batteries-included/archive/refs/tags/v3.7.2.tar.gz"
```
Comment (mseri) [inline packages/batteries/batteries.3.7.2/opam:46]: [ERROR] batteries.3.7.2: Checksum mismatch for https://github.com/ocaml-batteries-team/batteries-included/archive/refs/tags/v3.7.2.tar.gz:
          expected sha512=99a5afa3604c4cf0c849c670111d617f7f255acb0da043b73ddffdf0e299948bce52516ee31921f269de6088156c4e0a187e0b931543c6819c6b25966b303281
          got      sha512=c0f2a0fdc8253e0ea999d8d4c58bfbf32b18d251a2e1d9656bf279de5f01a33e9aabac3af4d95f465f8b671e7711ebd37218043face233340a0c11b08fa62f78

File: packages/batteries/batteries.3.7.2/opam [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  checksum: [
+    "md5=1fd7bddce07cf5d244fc9427f7b5e4d4"
+    "sha512=99a5afa3604c4cf0c849c670111d617f7f255acb0da043b73ddffdf0e299948bce52516ee31921f269de6088156c4e0a187e0b931543c6819c6b25966b303281"
```
Comment (kit-ty-kate) [inline packages/batteries/batteries.3.7.2/opam:49]:
```suggestion
    "sha512=c0f2a0fdc8253e0ea999d8d4c58bfbf32b18d251a2e1d9656bf279de5f01a33e9aabac3af4d95f465f8b671e7711ebd37218043face233340a0c11b08fa62f78"
```

File: packages/batteries/batteries.3.7.2/opam [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+]
+authors: ["OCaml batteries-included team"]
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ocaml-batteries-team/batteries-included"
+doc: "http://ocaml-batteries-team.github.io/batteries-included/hdoc2/"
+bug-reports:
+  "https://github.com/ocaml-batteries-team/batteries-included/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.02.3" & < "5.2"}
```
Comment (kit-ty-kate) [inline packages/batteries/batteries.3.7.2/opam:18]:
```suggestion
  "ocaml" {>= "4.05" & < "5.2"}
```
```
#=== ERROR while compiling batteries.3.7.2 ====================================#
# context              2.2.0~alpha4~dev | linux/x86_64 | ocaml-base-compiler.4.04.2 | pinned(https://github.com/ocaml-batteries-team/batteries-included/archive/refs/tags/v3.7.2.tar.gz)
# path                 ~/.opam/4.04/.opam-switch/build/batteries.3.7.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p batteries -j 71 @install
# exit-code            1
# env-file             ~/.opam/log/batteries-7-d75452.env
# output-file          ~/.opam/log/batteries-7-d75452.out
### output ###
# (cd _build/default && /home/opam/.opam/4.04/bin/ocamlc.opt -w -40 -w -3-32-50-52 -g -bin-annot -I src/.batteries_unthreaded.objs/byte -I /home/opam/.opam/4.04/lib/camlp-streams -I /home/opam/.opam/4.04/lib/num -intf-suffix .ml -no-alias-deps -o src/.batteries_unthreaded.objs/byte/batGc.cmo -c -impl src/batGc.pp.ml)
# File "src/batGc.pp.ml", line 1:
# Error: The implementation src/batGc.pp.ml
#        does not match the interface src/.batteries_unthreaded.objs/byte/batGc.cmi:
#        Values do not match:
#          external minor_words : unit -> (float [@unboxed])
#            = "caml_gc_minor_words" "caml_gc_minor_words_unboxed" [@@noalloc]
#        is not included in
#          external minor_words : unit -> (float [@unboxed])
#            = "caml_gc_minor_words" "caml_gc_minor_words_unboxed"
# (cd _build/default && /home/opam/.opam/4.04/bin/ocamlopt.opt -w -40 -w -3-32-50-52 -g -I src/.batteries_unthreaded.objs/byte -I src/.batteries_unthreaded.objs/native -I /home/opam/.opam/4.04/lib/camlp-streams -I /home/opam/.opam/4.04/lib/num -intf-suffix .ml -no-alias-deps -o src/.batteries_unthreaded.objs/native/batGc.cmx -c -impl src/batGc.pp.ml)
# File "src/batGc.pp.ml", line 1:
# Error: The implementation src/batGc.pp.ml
#        does not match the interface src/.batteries_unthreaded.objs/byte/batGc.cmi:
#        Values do not match:
#          external minor_words : unit -> (float [@unboxed])
#            = "caml_gc_minor_words" "caml_gc_minor_words_unboxed" [@@noalloc]
#        is not included in
#          external minor_words : unit -> (float [@unboxed])
#            = "caml_gc_minor_words" "caml_gc_minor_words_unboxed"
```

**Comment by mseri** [comment]
Looks fine, thanks

**Pull request merged**